spec/models/job_proxy_dispatcher_embedded_scan_spec.rb

```diff
@@ -83,7 +86,7 @@ def assert_at_most_x_scan_jobs_per_y_resource(x_scans, y_resource)
 
             it "should signal 2 jobs to start" do
               JobProxyDispatcher.dispatch
-              expect(MiqQueue.count).to eq(2)
+              expect(MiqQueue.where(:class_name => "Job").count).to eq(2)
```

Comment:
All events are put into the queue to be sent to automate.
Yep, we can delete all queue items before running dispatch to keep the test pure. That is another way to do it.